src/themes/custom/app/shared/dso-selector/modal-wrappers/dso-selector-modal-wrapper.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <div class="modal-header">{{'dso-selector.'+ action + '.' + objectType.toString().toLowerCase() + '.head' | translate}}
+    <button type="button" class="close" (click)="close()" aria-label="Close">
+      <span aria-hidden="true">×</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <h5 *ngIf="header" class="px-2">{{header | translate}}</h5>
+    <ds-dso-selector [currentDSOId]="dsoRD?.payload.uuid" [types]="selectorTypes" (onSelect)="selectObject($event)"></ds-dso-selector>
+  </div>
+</div>
```

Comment:
Rather than moving this file to the theme folder when it's not itself a themed component, I'd reference the version in the base theme as detailed in a comment above, and delete this file

Reply:
Yes, this file becomes useless.  Done.